templates/transaccion_completa/status-form.html

```diff
@@ -0,0 +1,37 @@
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <title>Ejemplos Webpay Plus - Solicitar Estado</title>
+    {% include 'include/header.html' %}
+  </head>
+
+  <body class="container">
+    <nav aria-label="breadcrumb">
+      <ol class="breadcrumb">
+        <li class="breadcrumb-item">
+          <a href="#">Ejemplos Webpay Plus - Solicitar Estado</a>
```

Comment:
same here, template name is for product `complete transaction` but this title is `Webpay plus`